### PR TITLE
Separate Link Quality name from LQI

### PR DESCRIPTION
**Pull request by chshu**
The Link Quality defined in Thread specification is different from the LQI defined in IEEE 802.15.4, separate their names to void confusing.

### CODE REVIEW

File: src/cli/README.md [hunk 1]
```diff
@@ -1383,10 +1383,10 @@ Print table of routers.
...
 ```bash
 > router table
-| ID | RLOC16 | Next Hop | Path Cost | LQI In | LQI Out | Age | Extended MAC     |
-+----+--------+----------+-----------+--------+---------+-----+------------------+
-| 21 | 0x5400 |       21 |         0 |      3 |       3 |   5 | d28d7f875888fccb |
-| 56 | 0xe000 |       56 |         0 |      0 |       0 | 182 | f2d92a82c8d8fe43 |
+| ID | RLOC16 | Next Hop | Path Cost | Link Quality In | Link Quality Out | Age | Extended MAC     |
++----+--------+----------+-----------+-----------------+------------------+-----+------------------+
+| 21 | 0x5400 |       21 |         0 |               3 |                3 |   5 | d28d7f875888fccb |
+| 56 | 0xe000 |       56 |         0 |               0 |                0 | 182 | f2d92a82c8d8fe43 |
```
Comment (turon) [inline src/cli/README.md]: Could we achieve a target of 80 columns or less?

Comment (chshu) [inline src/cli/README.md]: I agree that this output seems a litter longer, I will try to make it shorter. But is 80 columns a mandatory limitation? Or try to follow? It seems that some existing cli outputs exceed 80 columns, for example, the scan result.

Comment (chshu) [inline src/cli/README.md]: I have made the output shorter, the current length is 86 columns, which is the same with the [scan result](https://github.com/openthread/openthread/blob/master/src/cli/README.md#scan-channel).

File: src/cli/README.md [hunk 2]
```diff
@@ -208,10 +208,10 @@ Print table of attached children.
 
 ```bash
 > child table
-| ID  | RLOC16 | Timeout    | Age        | LQI In | C_VN |R|S|D|N| Extended MAC     |
-+-----+--------+------------+------------+--------+------+-+-+-+-+------------------+
-|   1 | 0xe001 |        240 |         44 |      3 |  237 |1|1|1|1| d28d7f875888fccb |
-|   2 | 0xe002 |        240 |         27 |      3 |  237 |0|1|0|1| e2b3540590b0fd87 |
+| ID  | RLOC16 | Timeout  | Age      | Link Quality | C_VN |R|S|D|N| Extended MAC     |
```
Comment (jwhui) [inline src/cli/README.md]: How about simply `LQ`, is that different enough from `LQI`?

Comment (chshu) [inline src/cli/README.md]: Updated, thanks.

**Pull request merged**